src/Calculator/App.xaml

```diff
@@ -93,7 +101,8 @@
                     <SolidColorBrush x:Key="AppControlPageTextBaseHighColorBrush" Color="{ThemeResource SystemColorWindowTextColor}"/>
                     <SolidColorBrush x:Key="AppControlPageTextRedColorBrush" Color="{ThemeResource SystemColorWindowTextColor}"/>
                     <SolidColorBrush x:Key="AppControlForegroundTransparentRevealBorderBrush" Color="{ThemeResource SystemColorButtonTextColor}"/>
-                    <SolidColorBrush x:Key="AppControlHighlightAltListAccentHighRevealBackgroundBrush" Color="{ThemeResource SystemColorHighlightTextColor}"/>
+                    <SolidColorBrush x:Key="AppControlHighlightAccentRevealBackgroundBrush" Color="{ThemeResource SystemColorHighlightColor}"/>
+                    <SolidColorBrush x:Key="AppControlHighlightAltListAccentHighRevealBackgroundBrush" Color="{ThemeResource SystemColorWindowColor}"/>
```

Comment:
```suggestion
                    <SolidColorBrush x:Key="AppControlHighlightListAccentHighRevealBackgroundBrush" Color="{ThemeResource SystemColorHighlightColor}"/>
```

Window color doesn't seem correct for a button...  The naming convention for these colors are described in [XAML theme resources](https://docs.microsoft.com/en-us/windows/uwp/design/controls-and-patterns/xaml-theme-resources#the-xaml-color-ramp-and-theme-dependent-brushes).

In the premade High Contrast schemes, button background and window background usually are the same color, so issues like this are difficult to spot.  I use a custom color palette for testing HC issues:
![image](https://user-images.githubusercontent.com/23107231/54965473-e5db7c00-4f2d-11e9-83d6-f046207b4739.png)

It's ugly but issues are _immediately_ apparent.  With the custom palette, you also want to avoid colors that are likely to be hardcoded, e.g. black/white.  I know I'm using those colors here, but it's not likely that a button has a hardcoded black/white style.